public/app/features/dashboard/api/publicDashboardApi.ts

```diff
@@ -38,6 +38,7 @@ export const publicDashboardApi = createApi({
         method: 'POST',
         data: params.payload,
       }),
+      extraOptions: { maxRetries: 0 },
```

Comment:
Why this is needed? What was the default value before this explicit 0?

Reply:
basically, the base query has 3 retries set, in order to make get request with 3 retries. In this case, I don't want this mutation query to have retries if it fails, because it could generate an unexpected behavior like automatically create the same public dashboard 2 times.